compiler/src/dotty/tools/dotc/printing/RefinedPrinter.scala

```diff
@@ -205,6 +205,11 @@ class RefinedPrinter(_ctx: Context) extends PlainPrinter(_ctx) {
     def toTextTuple(args: List[Type]): Text =
       "(" ~ argsText(args) ~ ")"
 
+    def toTextNamedTuple(elems: List[(TermName, Type)]): Text =
+      val elemsText = atPrec(GlobalPrec):
+        Text(elems.map((name, tp) => toText(name) ~ " : " ~ toText(tp)), ", ")
```

Comment:
it seems that tuple types ordinarily render wildcards as `?`, but named tuple does not:
```scala
scala> val x: (?,?) = (2,3)
val x: (?, ?) = (2,3)

scala> val x1: (a: ?, b: ?) = (2, 3)
val x1: (a : , b : ) = (2,3)
```